audit/projects/k8s-infra-ii-sandbox/services/enabled.txt

```diff
@@ -17,6 +17,7 @@ logging.googleapis.com               Cloud Logging API
 monitoring.googleapis.com            Cloud Monitoring API
 oslogin.googleapis.com               Cloud OS Login API
 pubsub.googleapis.com                Cloud Pub/Sub API
+secretmanager.googleapis.com         Secret Manager API
```

Comment:
@bernokl @hh I would be better to do this with the existing Terraform code we use for this project.

Reply:
Yeah I think it'd be good to try to get in the habit of using the terraform at https://github.com/kubernetes/k8s.io/tree/main/infra/gcp/clusters/projects/k8s-infra-ii-sandbox to update and manage your project

I opened https://github.com/kubernetes/k8s.io/issues/2279 as a followup issue for this

That said, I also don't see any secrets being used in this project? It might also be a good idea to disable services you're not using